msu/hooks/entity/tactical/humans/standard_bearer.nut

```diff
@@ -0,0 +1,11 @@
+// Is included during First QueueBucket
+// VANILLAFIX: http://battlebrothersgame.com/forums/topic/oncombatstarted-is-not-called-for-ai-characters/
+// This fix is spread out over 4 files: tactical_entity_manager, actor, player, standard_bearer
+::MSU.HooksMod.hook("scripts/entity/tactical/humans/standard_bearer", function(q) {
+	q.onInit = @(__original) function()
+	{
+		__original();
+		this.m.Skills.removeByID("perk.inspiring_presence");
+		::logWarning("MSU removed perk_inspiring_presence from standard_bearer. We do this to maintain unchanged vanilla behavior after our fix of onCombatStart for AI characters").
```

Comment:
Remove that warning though.

Reply:
The warning is there so that other modders using MSU are aware of this change as this is a gameplay change happening in MSU. Otherwise modders who expect that perk on standard_bearer and make changes to the perk will be surprised to see that the perk isn't present on those enemies.

Reply:
Okay, seems like would need to add a bunch more warnings then to account for all the stuff that we changed or fixed and unaware modders might not expect?

Reply:
This is different from other changes though. If we don't remove the `perk_inspiring_presence` from the standard_bearer that changes vanilla behavior. But changing perk setup of entities seems like something that is outside the scope of MSU so it seems like this is a kind of change no one expects MSU to make - which is why I put an explicit warning for it. Idk what do you think @Enduriel ?